### PR TITLE
Add soak workloads

### PR DESCRIPTION
**Pull request by alexeygorobets**
### What changes were proposed in this pull request?
Resolves [D2IQ-74327](https://jira.d2iq.com/browse/D2IQ-74327) 
Deploy KUDO Spark on soak and schedule periodic workloads

### Why are the changes needed?
To have Spark workloads running on soak cluster

### How were the changes tested?
Deployed on soak cluster 1.7

### CODE REVIEW

File: soaking/Makefile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.ONESHELL:
+SHELL := /bin/bash
+.SHELLFLAGS = -ec
+
+ROOT_DIR := $(CURDIR)
+
+NAMESPACE ?= spark-workloads
+
+.PHONY: install
+install:
```
Comment (akirillov) [inline soaking/Makefile:10]: should we include an "upgrade" plan too?

**Review by akirillov — COMMENTED** [review]
thanks @alexeygorobets, LGTM 👍  I have a question about the upgrade path. Shall we install the latest release version first and then trigger the upgrade to install the current repo version as a part of the `install` target? I think exercising the upgrade plan is the requirement for soak.

File: soaking/Makefile (hunk 1 above)
Comment (alexeygorobets) [reply soaking/Makefile:10]: thanks, made a separate make target for it

**Review by akirillov — APPROVED** [review]
thanks, @alexeygorobets. LGTM 👍

**Pull request merged**